src/omero_zarr/raw_pixels.py

```diff
@@ -78,26 +76,73 @@ def planeGen() -> np.ndarray:
         longest = longest // 2
         level_count += 1
 
-    field_groups = []
+    return add_raw_image(
+        planes=planes,
+        size_x=size_x,
+        size_y=size_y,
+        size_z=size_z,
+        size_c=size_c,
+        size_t=size_t,
+        d_type=d_type,
+        parent=parent,
+        level_count=level_count,
+        cache_dir=cache_dir,
+        cache_file_name_func=get_cache_filename,
+    )
+
+
+def add_raw_image(
+    *,
+    planes: Iterator[np.ndarray],
+    size_x: int,
+    size_y: int,
+    size_z: int,
```

Comment:
size_x and size_y are actually unnecessary.

Reply:
Good point! I've removed them.